### PR TITLE
Compatible with Zabbix 3.0 API

### PR DESCRIPTION
**Pull request by root-beer**
ref: https://skyarch.backlog.jp/view/NC-578

### CODE REVIEW

File: app/models/zabbix.rb [hunk 1]
```diff
@@ -546,6 +600,48 @@ def create_web_scenario(infra, web_scenario)
...
+      scenario_name = w.shift
+      if wh.has_key?(scenario_name)
+        wh[scenario_name] << w
+      else
+        wh[scenario_name] = [w]
+      end
+    end
+
+    # wh = {scenario_name => [[steps], [steps]]}
+    reqs = wh.each do |scenario_name, steps|
```
Comment (skyarch-networks) [inline app/models/zabbix.rb]: __[rubocop]__
* Lint/UselessAssignment: Useless assignment to variable - `reqs`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

**Comment by skyarch-networks** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[](https://www.sideci.com/github_repositories/31754199/auto_correct/rubocop/pull_request/261/branch/NC-578)

See the documentation if you want to get more information about auto-correct.
[Document](https://www.sideci.com/ja/docs/autofix-rubocop)

**Comment by root-beer** [comment]
Please review and merge PR. @Joeper214 :bow:

**Comment by Joeper214** [comment]
:+1: I'll review it on Monday.

**Comment by Joeper214** [comment]
LGTM! Thanks. @root-beer

**Comment by root-beer** [comment]
Thanks! :bow:

**Pull request merged**